### PR TITLE
virsh.py: Encrypt the password in set_user_password()

### PR DESCRIPTION
**Pull request by Grace-chhu**
Signed-off-by: Chenli Hu <chhu@redhat.com>

### CODE REVIEW

**Comment by Grace-chhu** [comment]
Test result with tp-libvirt PR#966:
# avocado run --vt-type libvirt virsh.set_user_password
...
TESTS      : 6
 (1/6) type_specific.io-github-autotest-libvirt.virsh.set_user_password.normal_test.root.encrypted: PASS (53.69 s)
 (2/6) type_specific.io-github-autotest-libvirt.virsh.set_user_password.normal_test.root.non-encrypted:PASS (54.54 s)
 (3/6) type_specific.io-github-autotest-libvirt.virsh.set_user_password.normal_test.normal_user.encrypted: PASS (56.66 s)
 (4/6) type_specific.io-github-autotest-libvirt.virsh.set_user_password.normal_test.normal_user.non-encrypted: PASS (56.19 s)
 (5/6) type_specific.io-github-autotest-libvirt.virsh.set_user_password.error_test.err_domain: PASS (35.84 s)
 (6/6) type_specific.io-github-autotest-libvirt.virsh.set_user_password.error_test.err_user: PASS (35.72 s)
RESULTS    : PASS 6 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0

**Comment by balamuruhans** [comment]
Looks good to me

File: virttest/virsh.py [hunk 1]
```diff
@@ -2710,6 +2710,11 @@ def set_user_password(domain=None, user=None, password=None,
     :return: CmdResult instance
     """
     cmd = "set-user-password"
+    if encrypted:
+        en_cmd = "openssl passwd -crypt %s" % password
```
Comment (will-Do) [inline virttest/virsh.py]: Just recall that the password could be ```None```

**Comment by Grace-chhu** [comment]
Closed as this is no longer need, encrypt the password in tp-libvirt

**Pull request closed without merging**